src/gtest/dict_test.cpp

```diff
@@ -0,0 +1,48 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
```

Comment:
Do we need gtest.h and gmock.h?  Or can we automatically include this in `generated_wrappers.hpp`?